flutter/test/sentry_flutter_util.dart

```diff
@@ -12,31 +11,54 @@ import 'mocks.dart';
 FutureOr<void> Function(SentryOptions) getConfigurationTester({
   bool isIOS = false,
   bool isWeb = false,
+  bool isAndroid = false,
 }) =>
     (SentryOptions options) async {
       options.dsn = fakeDsn;
 
       expect(kDebugMode, options.debug);
       expect('debug', options.environment);
 
-      expect(true, options.transport is FileSystemTransport);
+      expect(!isWeb, options.transport is FileSystemTransport);
 
       expect(
         options.integrations.whereType<FlutterErrorIntegration>().length,
         1,
       );
 
+      if (!isWeb) {
+        expect(
+          options.integrations.whereType<NativeSdkIntegration>().length,
+          1,
+        );
+      }
+
       if (isIOS) {
-        expect(5, options.integrations.length);
-      } else {
-        expect(4, options.integrations.length);
+        expect(
+          options.integrations.whereType<LoadContextsIntegration>().length,
+          1,
+        );
+      }
+
+      if (isAndroid) {
+        expect(
+          options.integrations
+              .whereType<LoadAndroidImageListIntegration>()
+              .length,
+          1,
+        );
       }
 
+      expect(
+        options.integrations.whereType<LoadReleaseIntegration>().length,
+        1,
+      );
+
       expect(sdkName, options.sdk.name);
       expect(sdkVersion, options.sdk.version);
       expect('pub:sentry_flutter', options.sdk.packages.last.name);
       expect(sdkVersion, options.sdk.packages.last.version);
 
-      expect('packageName@version+buildNumber', options.release);
-      expect('buildNumber', options.dist);
+      // expect('packageName@version+buildNumber', options.release);
+      // expect('buildNumber', options.dist);
```

Comment:
I will fix this PR as soon as I get an agreement of https://github.com/getsentry/sentry-dart/pull/214/files#r534165418